holidays/countries/suriname.py

```diff
@@ -0,0 +1,261 @@
+    observed_estimated_label = tr("Dag na de %s (geschat)")
+    start_year = 1972
+    supported_languages = ("en_US", "nl")
+
+    def __init__(self, islamic_show_estimated: bool = False, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChineseCalendarHolidays.__init__(self)
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=SurinameHinduHolidays)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=SurinameIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        dt = self._add_new_years_day(tr("Nieuwjaarsdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if (1981 <= self._year < 1993) or (2012 <= self._year < 2021):
+            # Day of Liberation and Renewal.
+            self._add_holiday_feb_25(tr("Dag van Bevrijding en Venieuwing"))
+
+        # Good Friday.
+        dt = self._add_good_friday(tr("Goede Vrijdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        # Easter Monday.
+        dt = self._add_easter_monday(tr("Tweede Paasdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year < 1976:
+            # Birthday of H.M. the Queen.
+            self._add_holiday_apr_30(tr("Verjaardag van H.M. de Koningin"))
+
+        # Labor Day.
+        dt = self._add_labor_day(tr("Dag van de Arbeid"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        # Day of Freedoms.
+        name = tr("Dag der Vrijheden") if self._year <= 2007 else tr("Keti Koti Dey")
+        dt = self._add_holiday_jul_1(name)
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year >= 2007:
+            # Indigenous People Day.
+            dt = self._add_holiday_aug_9(tr("Dag der Inheemsen"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Day of the Maroons.
+            dt = self._add_holiday_oct_10(tr("Dag der Marrons"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 1976:
+            # Republic Day.
+            name = "Dag van de Republiek" if self._year <= 2007 else tr("Onafhankelijkheidsdag")
+            dt = self._add_holiday_nov_25(tr(name))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        # Christmas Day.
+        dt = self._add_christmas_day(tr("Eerste Kerstdag"))
+        if self._year >= 2024:
+            self._add_observed(dt)
+
+        # Boxing Day.
```

Comment:
```suggestion
        # Second Day of Christmas.
```
The name "Boxing Day" is typical for countries whose culture is influenced by British culture; many European countries, including the Netherlands, name this holiday something like "Second Day of Christmas".